initials.c

```diff
@@ -0,0 +1,12 @@
+#include<stdio.h>
```

Comment:
Great job! The only thing I'd add is a comment at the top with your name and the function of this program since it's good practice to do so.